ddsreplayer/test/blackbox/mcap/dds/data_to_check.idl

```diff
@@ -0,0 +1,12 @@
+#include <iostream>
+
+
+struct DataToCheck
```

Comment:
I think this definition could go in HelloWorldSubscriber.h , it feels a bit odd to have it in a separated idl file.